src/types/user.ts

```diff
@@ -4,8 +4,8 @@
  */
 
 import * as express from "express";
-import { Either, isLeft, left, right } from "fp-ts/lib/Either";
-import { fromNullable, none, Option, some, tryCatch } from "fp-ts/lib/Option";
+import * as E from "fp-ts/lib/Either";
```

Comment:
I've seen sometimes fp-ts/xxx and sometime fp-ts/lib/xxx... can we take a choice?

Reply:
removed useless lib. thank you!